src/Services/FHIR/FhirPatientService.php

```diff
@@ -352,7 +354,7 @@ private function parseOpenEMRRaceRecord(FHIRPatient $patientResource, $race)
 
         $textExtension = new FHIRExtension();
         $textExtension->setUrl("text");
-        $textExtension->setValueString(new FHIRString($ombCategoryCoding->getDisplay()));
+        $textExtension->setValueString(new FHIRString($ombCategoryCoding->getDisplay()->getValue()->getValue()));
```

Comment:
Need to add some more helpers to Questionnaire Traits to validate and get value properties too make life easier...